src/tools/pub/pubspec.md

```diff
@@ -181,6 +189,24 @@ helps users understand where your package is coming from. If nothing else, you
 can always use the URL where you host the source code, such as
 [GitHub](https://github.com).
 
+### Repository
+
+The `repository` property should be a URL pointing to the repository for the
```

Comment:
Let's tighten this up a bit and refer to the homepage field. How about:

> This optional field should contain the URL for your package's source code — for example, `https://github.com/<user>/<repository>`. If you publish your package to pub.dartlang.org, then your package's page displays the repository URL. Also see the [homepage field](#homepage).

We should adjust the **Homepage** section as well to reflect how it interacts with `repository`.

@kevmoo, what should we say in the `homepage` section? Maybe change the last sentence from "If nothing else, you can always use the URL where you host the source code, such as GitHub." to something like this: "If nothing else, omit the [`repository` field](#repository) and set `homepage` to the URL where you host the source code."

Reply:
Actually, I don't think we care if people provide both `homepage` and `repository`, or just one of them... either way is probably fine :)

So let's add our plea to both places:
> While this is technically optional *please do* provide either a `homepage`
or a `repository`. It helps users understand where your package is coming from.